src/rss.py

```diff
@@ -58,8 +58,10 @@ def _fetch_company_tickers(
             "Company tickers file found and no refresh requested, skipping download ..."
         )
 
-def resolve_item_cik_and_ticker(item: Dict[str, Any], tickers_mapping: Dict[str, List[str]]) ->  Tuple[str, str, List[str]]:
 
+def resolve_item_cik_and_ticker(
+    item: Dict[str, Any], tickers_mapping: Dict[str, List[str]]
+) -> Tuple[str, str, List[str]]:
```

Comment:
Changes generated via `pipenv run format`, which calls the `black` formater.

I know that @opskov previously used `black` since it is mentioned in 762ddcd7cbb37bde52833874fc0008e76194c99a